build.gradle

```diff
@@ -481,12 +481,29 @@ dependencies {
     implementation 'commons-cli:commons-cli:1.3.1'
     implementation "org.bouncycastle:bcprov-jdk15to18:${versions.bouncycastle}"
     implementation 'org.ldaptive:ldaptive:1.2.3'
-    implementation 'io.jsonwebtoken:jjwt-api:0.10.8'
+
     implementation('org.apache.cxf:cxf-rt-rs-security-jose:3.5.5') {
         exclude(group: 'jakarta.activation', module: 'jakarta.activation-api')
     }
-    implementation 'com.github.wnameless:json-flattener:0.5.0'
-    implementation 'com.flipkart.zjsonpatch:zjsonpatch:0.4.4'
+    //JWT
+    implementation 'io.jsonwebtoken:jjwt-api:0.11.5'
```

Comment:
Do you think it is worth extracting these to a set version variable? It is only three so I don't have a preference but was not sure if you felt one way or the other.

Reply:
Sure I will do.